pyspark-notebook/Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN apt-get update --yes && \
 WORKDIR /tmp
 
 RUN if [ -z "${scala_version}" ]; then \
-    wget -qO "spark.tgz" "https://archive.apache.org/dist/spark/spark-${APACHE_SPARK_VERSION}/spark-${APACHE_SPARK_VERSION}-bin-hadoop${HADOOP_VERSION}.tgz"; \
+    wget -qO "spark.tgz" https://dist.apache.org/repos/dist/dev/spark/v3.3.2-rc1-bin/spark-3.3.2-bin-hadoop3.tgz; \
```

Comment:
Should we change the url to `dist.apache.org/...` in `main`?

Reply:
No. This is for testing spark 3.3.2 RC 1 
I will upgrade this PR when spark 3.3.2 is out.